platforms/hyperledger-indy/charts/indy-ledger-txn/values.yaml

```diff
@@ -0,0 +1,54 @@
+metadata:
```

Comment:
Please follow Values comments as in Fabric or Corda. One line explaining the item, and one line for a sample value.